enterprise/api/pagination.py

```diff
@@ -21,7 +21,8 @@ def get_paginated_response(data, request):
     Returns:
         (Response): DRF response object containing pagination links.
     """
-    url = request.build_absolute_uri().rstrip('/') + '/'
+    url = urlparse(request.build_absolute_uri())._replace(query=None).geturl()
```

Comment:
I think in general we should persist the query parameters in the original request to the next and previous urls, instead of completely replacing them, is it possible to recombine the url with a consolidation of the query parameters here and below?